pymeasure/instruments/instrument.py

```diff
@@ -101,6 +102,35 @@ def id(self):
         else:
             raise NotImplementedError("Non SCPI instruments require implementation in subclasses")
 
+    @property
+    def quick_properties_list(self):
+        return self.quick_props_list
+
+    @quick_properties_list.setter
+    def quick_properties_list(self, prop_list):
+        if type(prop_list) is str:
+            self.quick_props_list = [prop_list]
+        elif type(prop_list) is list:
+            self.quick_props_list = prop_list
+        else:
+            raise TypeError("Expected input of type str or list")
```

Comment:
AFAICT, the type-checking is the only thing that this property does beyond a simple attribute access. However, you're already nicely signaling that you expect a _list_, so I think we can expect users to assign a list (single-element if need be).
I would drop the property and use a plain attribute `quick_properties_list` initialised to `[]` as you already do.